packages/cozy-client/src/CozyClient.js

```diff
@@ -985,7 +985,9 @@ class CozyClient {
       throw new Error('Store is undefined')
     } else if (this.store && !force) {
       throw new Error(
-        'Client already has a store, it is forbidden to change store.'
+        `Client already has a store, it is forbidden to change store.
+         You might have called setStore too late ie you might have a dispatching 
+         a query before calling setStore`
```

Comment:
The error will contain the spaces of indentation, it'd be better to remove the indentation spaces for the 2nd and 3rd lines. 

I think here the problem is described but the solution could be put in a more explicit manner.
What do you think of 
"setStore must be called before any query is executed. Try to call setStore earlier in your code, preferably just after the instantiation of the client".